gazebo_plugins/include/gazebo_plugins/MultiCameraPlugin.hpp

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019 Open Source Robotics Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GAZEBO_PLUGINS__MULTICAMERAPLUGIN_HPP_
+#define GAZEBO_PLUGINS__MULTICAMERAPLUGIN_HPP_
+
+#include <string>
+#include <vector>
+
+#include "gazebo/common/Plugin.hh"
+#include "gazebo/sensors/MultiCameraSensor.hh"
+#include "gazebo/rendering/Camera.hh"
+#include "gazebo/util/system.hh"
+
+namespace gazebo
```

Comment:
Can we put this plugin in the `gazebo_plugins` namespace? Just in case someone creates the same plugin for Gazebo in the future.

In fact, feel free to make a pull request adding this plugin to [Gazebo 9](https://bitbucket.org/osrf/gazebo/src/gazebo9/plugins/) if you have time. It would be just a matter of copying this and the cc file over, and making a couple tweaks to follow [Gazebo's style](http://gazebosim.org/tutorials?tut=contrib_code&cat=development#Style) guide (which is different from ROS 2).

I can help you out with style changes for you since I'm used to it. Some things I can already see:

* rename `.hpp` to `.hh` and `.cpp` to `.cc`
* Use `camelCase` (i.e. `cameraNum`)
* Prefix all arguments with `_` (i.e. `_cameraNum`)
* There should be one `public` / `protected` / `private` per member
* `{` in a new line

The only problem is that even if we release this plugin for Gazebo 9.X, it won't be available to most ROS users, which use Gazebo 9.0.0 from Ubuntu. So we woudn't be able to use it in `gazebo_ros_pkgs`'s debian packages :confused:

Reply:
I will be making the PR soon